### ISSUE TITLE
Magnitude should be split up.

### ISSUE THREAD

**Issue by Leonid-Erlygin** [scikit-image/scikit-image#3565]
https://github.com/scikit-image/scikit-image/blob/bca527319521f0923856be56a73a542dc8aee4dc/skimage/feature/_hoghistogram.pyx#L72

"To reduce aliasing, votes are interpolated bilinearly between the neighbouring bin centres in both orientation and position"-from the original article

**Comment by stefanv** [comment]
Thank you for picking up on this subtlety, @Leonid-Erlygin.  Would you be willing to make a pull request to address the issue?

**Comment by Leonid-Erlygin** [comment]
> Thank you for picking up on this subtlety, @Leonid-Erlygin. Would you be willing to make a pull request to address the issue?

Yes, I can, but I don't have enough time(to fix it properly), because of exams. I will try to fix it in January, if no one would solve it earlier.
